CONTRIBUTING.md

```diff
@@ -74,14 +73,24 @@ The [bp-tf-ci.yaml](.cloudbees/workflows/bp-tf-ci.yaml) blueprints are orchestra
 
 ## Release
 
-Prerequisites:
+CloudBees CI Terraform EKS Addon versions try to be in sync with the [CloudBees CI releases](https://docs.cloudbees.com/docs/release-notes/latest/cloudbees-ci/).
 
-1. Project administrators must create a pull request from the `develop` branch to the `main` branch.
+Before a new CloudBees CI Helm chart is released, new features for this addon and its companion blueprints are merged into the integration branch (`develop`). When a new version of the CloudBees CI Helm chart is released, the addon is updated to the new version following this process:
 
-> [!IMPORTANT]
-> You must validate that the CasC bundle SCM tool is pointing to the `main` branch.
+1. Test the update in the integration branch (`develop`)
+   - Update The field `version` in CloudBees CI Terraform EKS Addon update needs to be updated to the new [version of the Helm chart](https://artifacthub.io/packages/helm/cloudbees/cloudbees-core/)
+   - The field `source` in the `eks_blueprints_addon_cbci` at blueprints folder must point to the local to the CloudBees CI Terraform EKS Addon the root of the repository `source = "../../"` (not to the remote [terraform registry version](https://registry.terraform.io/modules/cloudbees/cloudbees-ci-eks-addon/aws/latest))
+   - Test the update locally
+
+> [!TIP]
+> Use the following targets from [Makefile](Makefile): `deploy` > `validate` > `destroy`.
 
-2. Once the pull request is merged, verify that the `main` branch successfully passes the [Terraform CI build](#blueprint-terraform-ci-pipeline).
+2. Create a PR against the `main` branch including the Helm chart update plus other updates available in the integration branch (`develop`) ensuring the field `source` in the `eks_blueprints_addon_cbci` at blueprints is pointing to the remote [terraform registry version](https://registry.terraform.io/modules/cloudbees/cloudbees-ci-eks-addon/aws/latest) and `version >= "x.x.x"`.
```

Comment:
```suggestion
2. Create a PR against the `main` branch. including the Helm chart update plus other updates available in the integration branch (`develop`). Ensure that the field `source` in the `eks_blueprints_addon_cbci` at blueprints is pointing to the remote [terraform registry version](https://registry.terraform.io/modules/cloudbees/cloudbees-ci-eks-addon/aws/latest) and `version >= "x.x.x"`.
```
When you say "at blueprints" I'm not sure what you mean. Do you mean in the blueprints folder? If your user already knows this, ignore my comment. :)